docbook5-book/xml/common_intro_support.xml

```diff
@@ -135,7 +135,7 @@
    <listitem>
     <para>
      Technology previews are still in development.
-     Therefore, they may be functionally incomplete, unstable, or in other ways
+     Therefore, they may be functionally incomplete, unstable, or otherwise
```

Comment:
I'm not quite happy with this change, but I'll leave it up to you.

Reply:
I'm fine with the other changes above. With this one here, I'm also unsure, especially as we use the same default text block also within in our Release Note, see for example https://www.suse.com/releasenotes/x86_64/SUSE-SLES/15-SP4/index.html#intro-technology-preview.

@dariavladykina : What is your take on the suggested change above?

Reply:
The suggested change is grammatically correct and synonymous to "in other ways" which I think the editor wanted to avoid for its wordiness. It is also better to use it at the end of phrase, not as an adverbial modifier in front of a modified word.